Fix testing layers

gforcada opened the pull request:
I've been scratching my head over this for 3 days: without this fixes tests run fine but raise a `AttributeError: 'DB' object has no attribute '_mvcc_storage'` when tearing down layers:

```
Tearing down left over layers:
  Tear down der.freitag.testing.DerFreitagLayer:Integration in 0.000 seconds.
  Tear down der.freitag.testing.DerFreitagLayer in 0.033 seconds.
  Tear down collective.solr.testing.CollectiveSolr:Integration in 0.000 seconds.
  Tear down plone.app.contenttypes.testing.PloneAppContenttypes in 0.046 seconds.
  Tear down plone.app.event.testing.PAEventLayer Traceback (most recent call last):
  File "./bin/test", line 353, in <module>
    '--test-path', '/home/gil/Documents/freitag/git/zope-next/src/freitag.util.relations/src',
  File "/home/gil/.buildout/eggs/collective.xmltestreport-2.0.1-py2.7.egg/collective/xmltestreport/runner.py", line 66, in run
    failed = run_internal(defaults, args, script_parts=script_parts)
  File "/home/gil/.buildout/eggs/collective.xmltestreport-2.0.1-py2.7.egg/collective/xmltestreport/runner.py", line 79, in run_internal
    runner.run()
  File "/home/gil/.buildout/eggs/zope.testrunner-5.0-py2.7.egg/zope/testrunner/runner.py", line 191, in run
    self.run_tests()
  File "/home/gil/.buildout/eggs/zope.testrunner-5.0-py2.7.egg/zope/testrunner/runner.py", line 320, in run_tests
    tear_down_unneeded(self.options, (), setup_layers, True)
  File "/home/gil/.buildout/eggs/zope.testrunner-5.0-py2.7.egg/zope/testrunner/runner.py", line 764, in tear_down_unneeded
    l.tearDown()
  File "/home/gil/.buildout/eggs/plone.app.testing-6.1.3-py2.7.egg/plone/app/testing/helpers.py", line 393, in tearDown
    with zope.zopeApp() as app:
  File "/usr/lib64/python2.7/contextlib.py", line 17, in __enter__
    return self.gen.next()
  File "/home/gil/.buildout/eggs/plone.testing-7.0.3-py2.7.egg/plone/testing/zope.py", line 242, in zopeApp
    app = addRequestContainer(Zope2.app(connection), environ=environ)
  File "/home/gil/.buildout/eggs/Zope-4.1.3-py2.7.egg/Zope2/__init__.py", line 56, in app
    return bobo_application(*args, **kw)
  File "/home/gil/.buildout/eggs/Zope-4.1.3-py2.7.egg/App/ZApplication.py", line 75, in __call__
    connection = db.open()
  File "/home/gil/.buildout/eggs/ZODB-5.5.1-py2.7.egg/ZODB/DB.py", line 781, in open
    self._cache_size_bytes,
  File "/home/gil/.buildout/eggs/ZODB-5.5.1-py2.7.egg/ZODB/Connection.py", line 116, in __init__
    storage = db._mvcc_storage
AttributeError: 'DB' object has no attribute '_mvcc_storage'
```

gforcada committed:
fix layer teardown

When using the `FIXTURE` from `testing.py` the layer teardown fails
with:

`AttributeError: 'DB' object has no attribute '_mvcc_storage'`

gforcada committed:
Update CHANGES

hvelarde commented:
@gforcada check https://github.com/collective/collective.cover/commit/b790f1f082b079482fcd04fbcf15106676e99da3 to fix the error.

gforcada commented:
@hvelarde thanks I will try to update this PR today

gforcada committed:
fix: travis xvfb process not working

Taken from https://github.com/collective/collective.cover/commit/b790f1f082b079482fcd04fbcf15106676e99da3

Thanks to @hvelarde
